client/campusburn/src/app/login/page.tsx

```diff
@@ -58,12 +60,17 @@ export default function Page() {
             console.log("User not found in the database");
             toast({
                 title: "User not found",
-                variant: "destructive"
+                variant: "destructive",
+                className: "bg-red-600 text-white"
             });
         }
         else if( loginResponse.status === 200 ){
             console.log("Response after login: ", loginResponse);
-
+            toast({

```

Comment:
This toast is not required on login successful, as we are already pushing the user to a new route.

Reply:
Sure I'll update it